playbooks/appsemblerPlaybooks/monitoring.yml

```diff
@@ -17,12 +17,12 @@
     - role: "{{ appsembler_roles }}/logstash"
       when: "LOGSTASH_ENABLED|default(False)"
 
-    - role: "{{ appsembler_roles }}/stackdriver"
-      when: "cloud_provider == 'gcp'"
+#    - role: "{{ appsembler_roles }}/stackdriver"
+#      when: "cloud_provider == 'gcp'"
 
     - "{{ appsembler_roles }}/monit"
 
     - "{{ appsembler_roles }}/snort"
 
-    - role: newrelic
-      when: COMMON_ENABLE_NEWRELIC
+#    - role: newrelic
+#      when: COMMON_ENABLE_NEWRELIC
```

Comment:
I think we do still have a customer using New Relic. Do we absolutely need to disable this here?

Reply:
I had a bizarre syntax error, I'll give it another try. Otherwise there's nothing I have against using NewRelic.